src/epochs.lua

```diff
@@ -586,6 +572,7 @@ function epochs.computeTotalEligibleRewardsForEpoch(epochIndex, prescribedObserv
 		end
 	end
 	return {
+		totalEligibleGateways = #activeGatewayAddresses,
```

Comment:
The function `getActiveGatewaysBeforeTimestamp` could use some luadocs to indicate that an array is returned (and, thus, that # is safe to use here).

Reply:
added doc in https://github.com/ar-io/ar-io-network-process/pull/353/commits/11b9e1c33bb0975520a93ec5193b074670982a3f